tools/python3/git.py

```
@@ -0,0 +1,35 @@
+
+
+def log(N=10, use_short=False):
+    # https://git-scm.com/docs/pretty-formats
+    short = '--pretty=format:"%h%x09%an%x09%ad%x09%s"' if use_short else ""
+    return decode_bytes(run(f"git log -{N} {short}").stdout)
+
+
+def branch_exists(branch):
+    try:
+        run(f"git show-branch {branch}", check=True)
+    except subprocess.CalledProcessError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)